### ISSUE TITLE
Object Browser - Object's Member Description from tB DLL Double-Posted

### ISSUE THREAD

**Issue by GCuser99** [twinbasic/twinbasic#1415]
**Describe the bug**
When viewing a referenced tB ActiveX Dll object model in VBA's Object Browser, member descriptions from both @'Description annotation and VB_Description attribute are displayed...

**To Reproduce**

1. Create a tB ActiveX DLL project
2. Import a VBA class that has been RubberDuck-ized with Attribute Descriptions (see code below)
3. Build the DLL
4. Reference DLL from a VBA project
5. View the object model in VBA's Object Browser

**Expected behavior**
Should only see one description for an annotated member, instead of two.

**Screenshots**
![Screenshot 2022-12-09 070100](https://user-images.githubusercontent.com/26237126/206720760-959b0f1e-e1e4-4890-a70c-a7b5f6628a3e.png)
![Screenshot 2022-12-09 070029](https://user-images.githubusercontent.com/26237126/206721062-75c6185e-b77f-4062-9020-5914acc13f9f.png)
**Desktop:**
 - OS: Windows 11
 - Excel 365 (64-bit)
 - twinBASIC compiler version IDE Beta 201

**Additional context**
Note that the double-posting happens whether or not the @'Annotation and VB_Description attribute values are equal.

One fix could be an option in Project Properties to use either @'Annotation or VB_Description attribute for Object Browser description?

Another fix - ignore one or the other if they both exist? My preference would be to ignore the VB_Description.

**Comment by Greedquest** [comment]
This will happen often when importing code written in RD, so we cannot raise an error. Definitely a compiler warning of duplicate definition in same scope. 

Although I too prefer RD annotations, I think you have to rank the attribute statement higher and treat the annotation as a comment, if there is an attribute present, in order to be fully compatible with VBx.

**Comment by GCuser99** [comment]
Yep - giving one "rank" over the other would be perfect from my perspective. And that could be implemented with a Project Properties (or IDE Options?) toggle...

**Comment by retailcoder** [comment]
FWIW the solution picked here will likely end up implemented similarly in RD3, where hidden attributes are surfaced into the editor, with possibly conflicting values.
In RD 2.x the annotation comment is prioritized, because the comment is more readily accessible than the attribute in the VBIDE and so Rubberduck gets to show/consume the description content regardless of whether the corresponding attributes are synchronized or not.

**Comment by GCuser99** [comment]
@Greedquest I'd be ok working around whatever prioritization is decided, whether I have a choice or not. I don't think they BOTH should be posted as in the current state.

**Comment by Greedquest** [comment]
@retailcoder The difference is, the user actively chose to install RD and prefer its annotations to VBx attributes. With tB you don't know the user preference, but you do know backwards compatibility is king and someone somewhere might be relying on the fact that a comment is a comment and the attribute might not match but there's a very good reason for it... IDK I feel like I'm talking myself out of it, descriptions are cosmetic and RD is prettier:) But no, 100% compatibility goal.

**Comment by GCuser99** [comment]
> @retailcoder The difference is, the user actively chose to install RD and prefer its annotations to VBx attributes.

For a VBA'er like me using RD, there is no "preference" for RD annotations over VB description attributes - the annotation is just a convenient way of setting the underlying (and hidden in VBA) attribute value. RD will warn me if the two are different in order to reconcile, but the object browser only cares about the attribute value.

**Comment by Greedquest** [comment]
@GCuser99 I was talking about RD3. In RD2 (current) as you say sync is optional, so it's not really any different to vanilla VBA, just more convenient. 

In RD3 both the attributes and annotations will be visible in the brand new editor, and all the tooltips / places the documentation show up will be controlled by RD. In case of a clash then, RD has to pick which description it wants to use for the tooltip. I was arguing in that situation it can ignore what the normal VBE object browser says and just use its own annotations to drive the documentation, tooltips etc. Since the user installed RD3 they know what they're getting into. For tB however, comments should be comments, for full compatibility, and similar to RD2, may be used to _suggest_ a quickfix but nothing more.
